packages/stream_video_flutter/CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+## 0.9.4
+
+🐞 Fixed
+* Resolved an issue where the screen share track would disappear when zoomed in.
+* Fixed screen sharing layout issues on wide screens.
```

Comment:
I would add:
```suggestion
* Fixed screen sharing layout issues on wide screens.
* Fixed `ScreenShareCallParticipantsContent` with Spotlight view on very wide windows.
```

Because it can impact users